docs/for-devs/console/usermgnt.md

```diff
@@ -0,0 +1,33 @@
+---
+slug: user-management
+title: Teams - User Management
```

Comment:
`Team User Management` or `Team - User Management` for consistency with other team pages.